files/default/neutron-ha-tool.py

```diff
@@ -22,6 +22,7 @@
 import argparse
 import random
 import time
+import retrying
```

Comment:
List of imports is supposed to be sorted.

That being said, `retrying` is maybe not in the standard library? If not, then it needs a blank like before the `import`.